tests/PluginRegisterTest.php

```diff
@@ -71,9 +108,21 @@ public function testGetPluginsCSS()
    * @covers PluginRegister::filterPluginsByName
    */
   public function testGetPluginsCSSWithName()
+  {
+    $this->assertEquals($this->mockpr->getPluginsCSS()['test-plugin1'],
+                        array('plugins/test-plugin1/stylesheet.css'));
+  }
+
+  /**
+   * @covers PluginRegister::getPluginCallbacks
+   * @covers PluginRegister::filterPlugins
+   * @covers PluginRegister::filterPluginsByName
+   */
+  public function testGetPluginCallbacks()
   {
     $plugins = new PluginRegister();
-    $this->assertEquals(array('test-plugin' => array('plugins/test-plugin/stylesheet.css')), $this->mockpr->getPluginsCSS(array('test-plugin')));
+    $this->assertEquals($this->mockpr->getPluginCallbacks()['test-plugin1'],
```

Comment:
Here passing a $names parameter to getPluginCallbacks would increase test coverage.